config/routes.rb

```diff
@@ -0,0 +1,21 @@
+Rails.application.routes.draw do
+  resources :posts
+  devise_for :users
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/users/all', to: 'user#all', as: :find_friends
+  get '/users/:id', to: 'user#show', as: :user
+  get '/notifications/', to: 'notification#index', as: :notifications
+  get '/privacy_policy', to: 'static_pages#privacy', as: :privacy
+
+  match "*path", to: "application#render_404", via: :all
+
+  devise_scope :user do
+    authenticated :user do
+      root 'home#index', as: :authenticated_root
+    end
+
+    unauthenticated do
+      root 'home#welcome', as: :unauthenticated_root
```

Comment:
Home#welcome seems to be a log in page 🤔 

Why not to change app/views/devise/session/new.html  and use it here?

Reply:
http://localhost:3000/users/sign_in works ;)

Reply:
I guess  this comment https://github.com/talihomz/FakeBook/pull/3/files/d77418cd7b96494ff3f409df2db4960ae8d43cdb#r187860074 
was about this line of code.

>We wanted to follow Facebook's example of having the signup/login page as the home page. This means we had to change a few default routes to match this specific case.

You mean, you want  http://localhost:3000 (or www.fakebook.com) to
- open signin/up form for unauthenticated users
- open home index for authenticated users,

right?

Still my question is valid? Why do you need to create new action `home#welcome` instead of using `session#new` ?

Reply:
@talihomz @SavaVuckovic ☝️

Reply:
We have fixed this :)